app/controllers/passkit/api/v1/registrations_controller.rb

```diff
@@ -48,8 +48,10 @@ def show
         # @return If the request is not authorized, returns HTTP status 401.
         # @return Otherwise, returns the appropriate standard HTTP status.
         def destroy
-          registrations = @pass.registrations.where(passkit_device_id: params[:device_id])
-          registrations.delete_all
+          render json: {}, status: :ok unless @device
+
+          registrations = @pass.registrations.where(passkit_device_id: @device.id)
+          registrations.destroy_all
           render json: {}, status: :ok
```

Comment:
Double render happens here, you need to return after the first `render`.

Reply:
[Initial changes](https://github.com/max-si-m/passkit/blob/cc0f14ee20707c21f16ee7892cb31ee4603e83ab/app/controllers/passkit/api/v1/registrations_controller.rb#L50)

good point
will be fixed